go/tools/builders/stdlib.go

```diff
@@ -170,5 +171,21 @@ You may need to use the flags --cpu=x64_windows --compiler=mingw-gcc.`)
 	if err := goenv.runCommand(installArgs); err != nil {
 		return err
 	}
+
+	// Install the "cmd/pack" command from source if it's missing as it's no longer shipped with Go distributions
+	// after go1.25 https://github.com/golang/go/issues/74080
+	if _, err := os.Stat(filepath.Join(output, "pkg/tool", runtime.GOOS + "_" + runtime.GOARCH, "pack")); err != nil && os.IsNotExist(err) {
+		originalOS, originalARCH := os.Getenv("GOOS"), os.Getenv("GOARCH")
+		os.Setenv("GOOS", runtime.GOOS)
+		os.Setenv("GOARCH", runtime.GOARCH)
+		defer func() {
+			os.Setenv("GOOS", originalOS)
+			os.Setenv("GOARCH", originalARCH)
+		}()
+		toolArgs := goenv.goCmd("install", "-toolexec", abs(os.Args[0])+" filterbuildid", "cmd/pack")
```

Comment:
This may not work correctly in general: We build `pkg/tool` as part of the `stdlib` rule, but the execution platform of that rule may not match the execution platform of the Go rules that need run `pack`. 

Instead, I think that we need to duplicate (or extract) most of the `stdlib` rule (no need to split out `actions`) and make this new rule a `cfg = "exec"` dependency of each Go rule.

Reply:
ah, good point.

Reply:
Thanks, that works. I'll send it an updated PR soon, probably tomorrow.